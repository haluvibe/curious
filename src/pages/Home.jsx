import React from "react";
import { translate } from "react-i18next";
import { compose, pure } from "recompose";

const HomePage = ({ t }) => <div>{t("name")}</div>;

const enhance = compose(translate(), pure);

export default enhance(HomePage);
