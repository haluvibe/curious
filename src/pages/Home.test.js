import React from "react";
import toJson from "enzyme-to-json";
import { shallow } from "enzyme";
import Home from "./Home";
import i18n from "../localization/i18next";

test("Home component should render as expected", () => {
    const component = shallow(<Home />),
        tree = toJson(
            component
                .dive()
                .dive()
                .dive()
        );
    expect(tree).toMatchSnapshot();
});
