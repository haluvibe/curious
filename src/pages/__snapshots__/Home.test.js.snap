// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should render as expected 1`] = `
<div>
  Name
</div>
`;
